src/test/java/com/bina/varsim/tools/evaluation/VCFCompareTest.java

```diff
@@ -54,8 +54,8 @@ public void universalTestMethod(String directory, String[] additionalArgs) throw
     assertTrue(FileUtils.contentEquals(outputTruePositive.toFile(), new File(expectedTruePositive)));
     assertTrue(FileUtils.contentEquals(outputUnknownFalsePositive.toFile(), new File(expectedUnknownFalsePositive)));
     assertTrue(FileUtils.contentEquals(outputUnknownTruePositive.toFile(), new File(expectedUnknownTruePositive)));
-
-    outputJson.toFile().renameTo(new File(directory, "test_report.json"));
+    //move json file to test dir
```

Comment:
Can remove this comment and the commented out code since the stuff has already been moved.